src/containers/BooksList.js

```diff
@@ -1,14 +1,21 @@
 import React from "react"
+import { connect } from "react-redux"
 import Book from "../components/Book"
 import "bootstrap/dist/css/bootstrap.min.css"
-import { bookObj } from "../pages"
+import store from "../reducers/books"
+import { createBook, removeBook } from "../actions"
 
 const BooksList = () => {
-  const displayBooks = bookObj.map(book => {
+  const allBooks = store.getState().reducer
+  const displayBooks = allBooks.map(book => {
+
     return <Book key={book.id} book={book} />
   })
 
   return <div>{displayBooks}</div>
 }
 
-export default BooksList
+export default connect(state => ({ state }), { createBook, removeBook })(
+  BooksList
+)
+
```

Comment:
## Component Implementation 
* [ ] This component should be implementing the `handleBookRemove` [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)
* [ ] It should be connecting the `REMOVE_BOOK` action
* [ ] It should also be passing the `handleRemoveBook` to Book component [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)

#### Kindly Implement these changes as they are mandatory, team!